homepage/migrations/0003_course_test_data.py

```diff
@@ -0,0 +1,38 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('homepage', '0001_initial'),
```

Comment:
maybe it will be good to update it after 
https://github.com/beyond-io/ClassRater/pull/29 will merge

Reply:
For the user downloading this repository, yes, but this file is independent of the user file completely
I could change the numbering to 0002 to discourage loss of data \ manage.py ignoring the middle migration, if that would work better?
But then #29 would have to change the number as well

Reply:
never mind you can leave it like that
if needed we can change it in other PR